src/document-converter.ts

```diff
@@ -304,7 +304,7 @@ export class DocumentConverter {
     return {
       originalUrl: this.originalUrl,
       convertedUrl: this.convertedUrl,
-      convertedFilePath: getJsModuleConvertedFilePath(this.originalUrl),
+      convertedFilePath: this.convertedUrl as any as ConvertedDocumentFilePath,
```

Comment:
this breaks conversion:

```
{
  originalUrl: 'workspace/paper-button/paper-button.html',
  convertedUrl: 'workspace/@polymer/paper-button/paper-button.js',
  convertedFilePath: 'workspace/paper-button/paper-button.html',
}
```

we need to know where the file will exist as an npm dependency, that's what ConvertedDocumentUrl gets us. But we also need to know where to write the file, and that's what `ConvertedDocumentFilePath` gets us.